src/frontend/js/order.js

```diff
@@ -3,6 +3,35 @@ $(function() {
     // Javascript of the order application.
     // **************************************
 
+    if ($('.ui.order-override.modal')) {
+      $('.ui.order-override.modal').modal('show');
```

Comment:
The idea is avoid to load the same HTML element by `jQuery` more than one time, with that, we will avoid to force to `jQuery` to go again through the XHTML. 
Some time when the HTML element is modified, we need to reload it, but if we are not in that situation, we can assign the value of $('.ui.order-override.modal') to a new `var` and use the new `var` instead of repeat the `jQuery` expression.
There are more case in this file... `$('#id_delivery_dates')`, `$('#id_override_dates')`, `$('#reason_other_field textarea')` and others...
I know that this changes does not affect the result, but we can get a more beautiful code :)